apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/RDASinkTest.java

```diff
@@ -0,0 +1,52 @@
+package gov.cms.bfd.pipeline.rda.grpc;
+
+import java.io.IOException;
+import java.util.Arrays;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class RDASinkTest {
```

Comment:
Naming convention thing:

```suggestion
public class RdaSinkTest {
```